src/feedback.js

```diff
@@ -0,0 +1,45 @@
+import { gameState } from "./index.js";
+import { clearCardsRow } from "./navigation.js";
+import { getPromiseFromEvent } from "./game.js";
+
+async function showFeedback() {
+  clearCardsRow();
+  document.querySelector(".start-button").classList.add("hidden");
+
+  const feedbackView = document.createElement("div");
+  feedbackView.classList.add("feedback-view");
+
+  const score = document.createElement("h3");
+  const emoji = document.createElement("span");
+  emoji.classList.add("emoji");
+
+  feedbackView.append(score, emoji);
+  document.querySelector("main").append(feedbackView);
+
+  if (gameState.mistakes) {
+    score.textContent = `Oh no! ${gameState.mistakes} mistakes :(`;
+    emoji.textContent = "😢😢😢";
+    new Audio("./data/audio/failure.mp3").play();
```

Comment:
`Audio().play()` is the `async` method, so we should use it with the `await` command.